docs/architecture/cloud-native/azure-data-storage.md

```diff
@@ -1,210 +0,0 @@
----
```

Comment:
do you need to redirect these articles you are removing somewhere if customers have they bookmarked?

Reply:
Sorry, not sure I understand the question.